greedybear/cronjobs/base.py

```diff
@@ -31,9 +31,19 @@ def _base_search(self, honeypot):
         search = Search(using=self.elastic_client, index="logstash-*")
         self.log.debug(f"minutes_back_to_lookup: {self.minutes_back_to_lookup}")
         gte_date = f"now-{self.minutes_back_to_lookup}m/m"
-        search = search.filter(
-            "range", **{"timestamp": {"gte": gte_date, "lte": "now/m"}}
+        # Some honeypots have different column for the time
```

Comment:
cool, did you test it with a local T-POT?

Reply:
Yes, tested it on a local VM with Greedybear on the same machine